loops-and-arrays.js

```diff
@@ -12,25 +12,44 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(names) geeft: ["Henkje", "Pietje", "Fredje", "Joopje"]
 
+console.log(names);
+
+for (let i = 0; i < names.length; i++) {
 
+    names[i] = names[i] + "je";
+}
 
+console.log(names);
 
 // ==========================================
 // Opdracht 2
 // Schrijf een script dat ieder EVEN getal in onderstaande array met 2 vermenigvuldigd, en ieder ONEVEN getal met 3
 // Zorg ervoor dat dit ook zou werken als de array wel 100 of 200 getallen zou bevatten!
 // ==========================================
-
+console.log(" ================= Opdracht 2 =========================")
 const numbers = [2, 4, 5, 29, 38];
 
+console.log(numbers);
 // Verwachtte uitkomsten:
 // Vóór het script zie je de originele waardes:
 // console.log(numbers) geeft: [2, 4, 5, 29, 38];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(numbers) geeft: [ 4, 8, 15, 87, 76 ];
 
+for (let i = 0; i < numbers.length; i++) {
+    
+    if (numbers[i] % 2 !== 0) {
+        // Oneven getal uitkomst
+        numbers[i] = numbers[i] * 3;
 
```

Comment:
Goed gedaan idealiter zou je twee keer de conditie checken of iets waar is dus twee keer if.